docs/pages/index.md

```diff
@@ -0,0 +1,51 @@
+3. Load a custom dataset like [SWO Ecoplot](api/datasets/swo_ecoplot.md) (or bring your own).
+```python
+from sknnr.datasets import load_swo_ecoplot
+
+X, y = load_swo_ecoplot(return_X_y=True, as_frame=True)
+```
+4. Train, predict, and score [as usual](https://scikit-learn.org/stable/getting_started.html#fitting-and-predicting-estimator-basics).
+```python
+from sklearn.model_selection import train_test_splot
+
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+est = est.fit(X_train, y_train)
+est.score(X_test, y_test)
+```
+5. Check out the additional features like [independent scoring](usage.md/#independent-scores-and-predictions), [dataframe indexing](usage.md/#retrieving-dataframe-indexes), and [dimensionality reduction](usage.md/#dimensionality-reduction).
+```python
+# Evaluate the model using the second-nearest neighbor in the test set
+print(est.fit(X, y).independent_score_)
+
+# Get the dataframe index of the nearest neighbor to each plot
+print(est.kneighbors(return_dataframe_index=True, return_distance=False))
+
+# Apply dimensionality reduction using CCorA ordination
+MSNRegressor(n_components=3).fit(X_train, y_train)
+```
+
+## Acknowledgements
+
+`sknnr` was inspired by the [yaImpute](https://cran.r-project.org/web/packages/yaImpute/index.html) package for R (Crookston & Finley 2008). Thanks to Andrew Hudak for the inclusion of the [Moscow Mountain / St. Joes dataset](api/datasets/moscow_stjoes.md) (Hudak 2010), and Tom DeMeo for the inclusion of the [SWO Ecoplot dataset](api/datasets/swo_ecoplot.md) (Atzet et al., 1996). Development of this package was funded in part by an appointment to the United States Forest Service (USFS) Research Participation Program administered by the Oak Ridge Institute for Science and Education (ORISE) through an interagency agreement between the U.S. Department of Energy (DOE) and the U.S. Department of Agriculture (USDA).
+
+## References
+
+- Atzet, T, DE White, LA McCrimmon, PA Martinez, PR Fong, and VD Randall. 1996. Field guide to the forested plant associations of southwestern Oregon. USDA Forest Service. Pacific Northwest Region, Technical Paper R6-NR-ECOL-TP-17-96.
+- Crookston, NL, Finley, AO. 2008. yaImpute: An R package for kNN imputation. Journal of Statistical Software, 23(10), 16. 
+- Hudak, A.T. 2010. Field plot measures and predictive maps for "Nearest neighbor imputation of species-level, plot-scale forest structure attributes from LiDAR data". Fort Collins, CO: U.S. Department of Agriculture, Forest Service, Rocky Mountain Research Station. https://www.fs.usda.gov/rds/archive/Catalog/RDS-2010-0012.
+- Moeur M, Stage AR. 1995. Most Similar Neighbor: An Improved Sampling Inference Procedure for Natural Resources Planning. Forest Science, 41(2), 337–359.
+- Ohmann JL, Gregory MJ. 2002. Predictive Mapping of Forest Composition and Structure with Direct Gradient Analysis and Nearest Neighbor Imputation in Coastal Oregon, USA. Canadian Journal of Forest Research, 32, 725–741.
```

Comment:
I don't love the duplication between this and the readme... We could put them in a separate page and link them from the readme, but it feels important to have them upfront. Let me know if you have any ideas.

Reply:
I just found mkdocs/mkdocs#2879 which points out that snippets can be used to copy the contents of the `README.md` into `index.md`, which would allow us to share the same content without any duplication. I think we would still have to use absolute links to the docs pages in the README[^1] and avoid any platform-specific formatting[^2], but if we do want the same content in both locations, this looks like the best approach.

[^1]: e.g. `https://sknnr.readthedocs.io/api/estimators/gnn` instead of `api/estimators.gnn.md`.
[^2]: e.g. the Github `[!WARNING]` tag or mkdocs [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage).

Reply:
This looks like a cool approach.  Does it make sense to keep editing on `README.md` and then just using that link for `index.md`?  I obviously haven't done that for this round.  

I'm not totally clear how we avoid the platform-specific formatting.  If `README.md` will get rendered on Github using `[!WARNING]` syntax, but `index.md` will get rendered on RTD using `!!! warning` syntax, can we really use the snippet from `README.md` in `index.md`.  Sorry for the naive question.

Reply:
> This looks like a cool approach. Does it make sense to keep editing on README.md and then just using that link for index.md?

I think I'll copy everything from `index.md` into `README.md` now and link them so we don't have to worry about managing both.

> If README.md will get rendered on Github using [!WARNING] syntax, but index.md will get rendered on RTD using !!! warning syntax, can we really use the snippet from README.md in index.md.

We'll just have to not use either and stick with some vanilla markdown that both can render. Since the only offender is the "in development" warning, I think that'll be fine if we just throw an emoji and some bold text in there.

Reply:
`index.md` now copies all content from `README.md`, so no more duplication! I tried to replace all the relative links with absolute links to where the final docs will be served, but we'll have to double check once that's actually live.

Reply:
> We'll just have to not use either and stick with some vanilla markdown that both can render. Since the only offender is the "in development" warning, I think that'll be fine if we just throw an emoji and some bold text in there

This looks great.  I had confused myself as most of the admonitions were in `usage.md` and those can stay the same because we're not pulling content from some other file.